src/_locales/en/messages.json

```diff
@@ -0,0 +1,26 @@
+{
+    "extensionName": {
+        "message": "Facebook Container",
+        "description": "Name of the extension."
+    },
+    "extensionDescription": {
+        "message": "Facebook Container isolates your Facebook activity from the rest of your web activity in order to prevent Facebook from tracking you outside of the Facebook website via third party cookies.",
+        "description": "Description of the extension."
+    },
+    "onFacebookHeader": {
+        "message": "You're browsing in a Facebook Container tab",
```

Comment:
We should use proper apostrophes in the strings `’`.

Reply:
This one? ’

Reply:
Yes